conu/apidefs/container.py

```diff
@@ -39,12 +39,19 @@ def __init__(self, image, container_id, name):
         :param container_id: str, unique identifier of this container
         :param container_id: str, pretty container name
         """
+        self.name = name
+        self._id = container_id
+
+        if not image:
+            image_name = self.get_image_name()
+            if image_name:
+                image = Image(image_name)
+
```

Comment:
I would add there `else:` because next check is redundant, from my point of view.

Reply:
You're right. Thanks.

Reply:
Done.